src/app/domains/Plan/components/lists/PlanSimpleList/PlanSimpleList.template.js

```diff
@@ -1,43 +1,42 @@
 import PropTypes from 'prop-types'
-import { List, Collapse } from 'antd'
-import { Box } from 'antd-styled'
-import { TechnologyAdvancedView } from 'domains/Technology/components/views'
-import { getGrid } from '~/utils'
+import { Collapse } from 'antd'
+import { TechnologyAdvancedList } from 'domains/Technology/components/lists'
 import { PlanAdvancedView } from 'domains/Plan/components/views'
+
+import { COLLECTIONS } from 'app/constants'
 const { Panel } = Collapse
 
 /**
  * @info PlanSimpleList (05 Mar 2021) // CREATION DATE
  *
  * @comment PlanSimpleList - React component.
  *
- * @since 15 Mar 2021 ( v.0.0.3 ) // LAST-EDIT DATE
+ * @since 16 Mar 2021 ( v.0.0.4) // LAST-EDIT DATE
  *
  * @return {ReactComponent}
  */
 
 const PlanSimpleList = (props) => {
   // [INTERFACES]
-  const { data } = props
+  const { plans, setActivePlanId } = props
+
+  // [HELPER_FUNCTIONS]
+  const onPlanChange = (planId) => {
+    setActivePlanId(planId)
+  }
 
   // [TEMPLATE]
   return (
-    <Collapse expandIconPosition="right" ghost accordion>
-      {data.map((plan) => (
-        <Panel header={<PlanAdvancedView planId={plan.id} />} key={plan.id}>
-          <List
-            grid={{
-              gutter: 16,
-              ...getGrid({ xs: 1 })
-            }}
-            dataSource={Object.keys(plan.technologyIds)}
-            renderItem={(technology) => (
-              <List.Item>
-                <Box p={10}>
-                  <TechnologyAdvancedView technologyId={technology} />
-                </Box>
-              </List.Item>
-            )}
+    <Collapse
+      expandIconPosition="right"
+      ghost
+      accordion
+      onChange={onPlanChange}>
+      {plans.map((plan) => (
+        <Panel header={<PlanAdvancedView plan={plan} />} key={plan.id}>
+          <TechnologyAdvancedList
+            refCollectionTechnologies={`${COLLECTIONS.PLANS}/${plan.id}/${COLLECTIONS.TECHNOLOGIES}`}
+            refCollectionMaterials={`${COLLECTIONS.PLANS}/${plan.id}/${COLLECTIONS.MATERIALS}`}
```

Comment:
Read previous 2 comments

Reply:
I can`t move this to  computed properties , because there is a loop for plans